utils/build-script-impl

```diff
@@ -917,7 +917,7 @@ function false_true() {
 CROSS_COMPILE_HOSTS=($CROSS_COMPILE_HOSTS)
 for t in "${CROSS_COMPILE_HOSTS[@]}"; do
     case ${t} in
-        iphone* | appletv* | watch* | linux-armv6 | linux-armv7 )
+        iphone* | appletv* | watch* | linux-armv6 | linux-armv7 | android-aarch64 )
```

Comment:
Can this be `android-*` like the ones at the start of the line? Thanks.

Reply:
I've never tried anything other than AArch64, but I guess the others would work too. I'll change it.